packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 48.13.0 - April 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Add check for local dev server start failure ([PR #2128](https://github.com/forcedotcom/salesforcedx-vscode/pull/2128))
```

Comment:
We can remove this piece - this was internal as part of the LWR switch from Talon. It's not something customers would have experienced yet